website/docs/index.html.markdown

```diff
@@ -9,107 +9,125 @@ description: |-
 
 # Azure Provider
 
-The Azure Provider is used to interact with the many resources supported by Azure Resource Manager (AzureRM) through its APIs.
+The Azure Provider can be used to configure infrastructure in [Microsoft Azure](https://azure.microsoft.com/en-us/) using the Azure Resource Manager API's. Documentation regarding the [Data Sources](/docs/configuration/data-sources.html) and [Resources](/docs/configuration/resources.html) supported by the Azure Provider can be found in the navigation to the left.
 
-~> **Note:** This supercedes the [legacy Azure provider](/docs/providers/azure/index.html), which interacts with Azure using the Service Management API.
+Interested in the provider's latest features, or want to make sure you're up to date? Check out the [changelog](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/CHANGELOG.md) for version information and release notes.
 
-Use the navigation to the left to read about the available resources.
+## Authenticating to Azure
 
-# Creating Credentials
+Terraform supports a number of different methods for authenticating to Azure:
 
-Terraform supports authenticating to Azure through a Service Principal or the Azure CLI.
+* [Authenticating to Azure using the Azure CLI](auth/azure_cli.html)
+* [Authenticating to Azure using Managed Service Identity](auth/managed_service_identity.html)
+* [Authenticating to Azure using a Service Principal and a Client Certificate](auth/service_principal_client_certificate.html)
+* [Authenticating to Azure using a Service Principal and a Client Secret](auth/service_principal_client_secret.html)
 
-We recommend [using a Service Principal when running in a shared environment](authenticating_via_service_principal.html) (such as within a CI server/automation) - and [authenticating via the Azure CLI](authenticating_via_azure_cli.html) when you're running Terraform locally.
+---
 
-~> **NOTE:** Authenticating via the Azure CLI is only supported when using a User Account. If you're using a Service Principal (e.g. via `az login --service-principal`) you should instead [authenticate via the Service Principal directly](authenticating_via_service_principal.html).
+We recommend using either a Service Principal or Managed Service Identity when running in automation (for example on a CI Server) - and authenticating using the Azure CLI when running Terraform locally.
```

Comment:
This wording confuses me "...running in automation...", maybe "...running Terraform via automation..." or something along those lines?

Reply:
using the combined comment from above